assets/out

```diff
@@ -100,10 +102,19 @@ elif [ -n "$build" ]; then
     docker_pull "${repository}:${cache_tag}"
   fi
 
+  expanded_build_args=""
+  if [ -n "$build_args" ]; then
+    expanded_build_args=$(echo $build_args|jq -r 'with_entries(.key |= "--build-arg " + . )|with_entries(.key = .key + "=" +.value)|keys|join(" ")')
```

Comment:
To be honest it's probably time to rewrite `out` in Go; it's been getting pretty complicated over time. I'm not sure if you wanna bite that bullet as part of this PR though. We've at least done that for `check` already.